src/elements/content-sidebar/activity-feed/app-activity/AppActivity.scss

```diff
@@ -0,0 +1,64 @@
+@import '../../../common/variables';
+
+.be {
+    .bcs-app-activity-container {
+        position: relative;
+        z-index: 1;
+
+        &:hover,
+        &.bcs-is-focused {
+            .bcs-inline-delete-container {
+                opacity: 1;
+            }
+        }
+
+        .bcs-app-activity {
+            display: flex;
+            margin-top: 23px;
+            opacity: .9999;
```

Comment:
Is this needed? Can it be `opacity: 1`?

Reply:
Trying to be consistent with Comment styling, since this is so similar. Removed it